### PR TITLE
Added information about availability of IB feature

### PR DESCRIPTION
**Pull request by anwara-ms**
Added information about availability of IB feature

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/security-compliance-overview.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ Teams has a wide range of information to help you with compliance areas, includi
 
 ### Information Barriers
 
-Information barriers are policies put in-place by Teams administrators to do things like keep people or groups from communicating with one another (when there is no business need for them to do so, or a regulatory reason to block them from doing so), and it also allows you to set policies relating to things like lookups and eDiscovery (covered below). These policies can impact users in 1:1 chats, group chats, or at a team-level.
+Information barriers are policies put in-place by Teams administrators to do things like keep people or groups from communicating with one another (when there is no business need for them to do so, or a regulatory reason to block them from doing so), and it also allows you to set policies relating to things like lookups and eDiscovery (covered below). These policies can impact users in 1:1 chats, group chats, or at a team-level. Information Barrier feature is available in public cloud and starting Jan'2021 it has been rolled out to GCC cloud as well.
```
Comment (JohanFreelancer9) [inline Teams/security-compliance-overview.md:97]:
```suggestion
Information barriers are policies put in-place by Teams administrators to do things like keep people or groups from communicating with one another (when there is no business need for them to do so, or a regulatory reason to block them from doing so), and it also allows you to set policies relating to things like lookups and eDiscovery (covered below). These policies can impact users in 1:1 chats, group chats, or at a team-level. The Information Barrier feature is available in the public cloud and starting January 2021 it has been rolled out to the GCC cloud.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@MicrosoftHeidi PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by anwara-ms** [comment]
Why is it taking so long. @SerdarSoysal ? Can you please give me access to publishing these things myself. This is a huge blocker in providing A1 service to customers. We are live in GCC for a long time, but the blog is not updated for over a month!

**Pull request merged**